src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/set_global_properties_request.cc

```diff
@@ -820,9 +831,134 @@ bool SetGlobalPropertiesRequest::IsWhiteSpaceExist() {
         }
       }
     }
+
+    if (msg_params[strings::keyboard_properties].keyExists(
+            hmi_request::customize_keys)) {
+      const smart_objects::SmartArray* custom_keys_array =
+          msg_params[strings::keyboard_properties][hmi_request::customize_keys]
+              .asArray();
+
+      auto it_keys = custom_keys_array->begin();
+      auto it_keys_end = custom_keys_array->end();
+
+      for (; it_keys != it_keys_end; ++it_keys) {
```

Comment:
Why didn't you use range-based for?

Reply:
@VladSemenyuk There is no good reason, only for consistency with similar checks above

Reply:
I think one line instead of three is a good enough reason.

Reply:
Maybe it makes sense to change loops above too. these modifications will make this function smaller and more readable.

Reply:
First suggestion is implemented in 5d8aa90